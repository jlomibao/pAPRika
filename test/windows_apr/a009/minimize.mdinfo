

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2750E+02     4.8971E-01     1.5810E+00     H32       105

 BOND    =        1.2644  ANGLE   =      107.5396  DIHED      =       52.6172
 VDWAALS =      -68.5953  EEL     =     1332.4267  EGB        =     -127.4359
 1-4 VDW =        5.5120  1-4 EEL =    -2130.8451  RESTRAINT  =        0.0190
 EAMBER  =     -827.5165
 NMR restraints: Bond =    0.004   Angle =     0.013   Torsion =     0.001
===============================================================================
