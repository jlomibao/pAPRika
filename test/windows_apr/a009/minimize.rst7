defa
  125
  -0.5298987   4.9567304   1.8185217   1.0065215   4.8939802   1.8190552
  -0.8905711   4.2784902   0.5808863  -0.8948536   4.2467759   3.0366881
  -0.9097561   5.9848392   1.8306263   1.4689916   5.8876650   1.8316165
   1.3117218   4.1886662   0.5818339   1.3119735   4.1562620   3.0370050
   2.6800168   4.0744022   0.0977596   2.6755299   4.0945257   3.5490614
   0.1930875   3.8268801   3.7571100   0.1965845   3.8829129  -0.1518668
   0.1731379   3.3296134   4.8697295   0.1784964   3.4158801  -1.2774602
   2.6640685   4.0227077  -0.9970086   3.2410930   4.9581194   0.4228490
   2.6390884   4.0654893   4.6441748   3.2160302   4.9883028   3.2167107
  -4.6625642   1.9869648   1.8299769  -3.8195544   3.2736701   1.8253020
  -4.2300000   1.2988309   0.6212227  -4.2775094   1.3413463   3.0779131
  -5.7396881   2.1886600   1.8061231  -4.4350648   4.1803185   1.8011899
  -3.0233573   3.1465374   0.6120793  -3.0669657   3.1837795   3.0704220
  -2.2629950   4.2764287   0.0947823  -2.2587152   4.2973010   3.5490340
  -3.3905220   2.0802962   3.8142578  -3.3047466   2.0087954  -0.0978980
  -3.0149557   1.8384319   4.9484261  -2.8727469   1.7219365  -1.2009196
  -2.2497866   4.2209555  -0.9998375  -2.7495987   5.2042692   0.4169226
  -2.2247201   4.2633654   4.6440852  -2.7234135   5.2333218   3.2183410
  -4.0957895  -3.0755997   1.8585773  -4.8242066  -1.7207027   1.8491307
  -3.3333009  -3.0526496   0.6166692  -3.2954457  -3.0155212   3.0744097
  -4.7871967  -3.9258175   1.8816372  -5.9149008  -1.8281823   1.8658665
  -4.3761701  -1.1108504   0.6040879  -4.3409138  -1.0721464   3.0605880
  -4.9527715   0.1362803   0.1217328  -4.9631040   0.1488903   3.5561855
  -3.4816448  -1.8575965   3.7831086  -3.5511654  -1.9234228  -0.1267098
  -3.0301369  -1.6062662   4.8869920  -3.1449977  -1.7120607  -1.2564257
  -4.9153547   0.1420481  -0.9737132  -5.9930560   0.1957753   0.4618926
  -4.9330866   0.1398434   4.6518371  -6.0025016   0.1797754   3.2095750
   0.5781884  -5.0956134   1.8722048  -0.9582288  -5.0315641   1.8721689
   0.9426303  -4.3906850   0.6510526   0.9404807  -4.4108215   3.1058097
   0.9576892  -6.1238961   1.8644712  -1.4219906  -6.0246981   1.8646995
  -1.2630251  -4.2989148   0.6510175  -1.2620585  -4.3188192   3.1057292
  -2.6264678  -4.2337089   0.1404576  -2.6297539  -4.2035792   3.5921451
  -0.1458837  -4.0078786   3.8358621  -0.1445409  -3.9687343  -0.0688360
  -0.1264789  -3.5309796   4.9572794  -0.1242803  -3.4676590  -1.1797808
  -2.5914496  -4.2063569  -0.9547462  -3.1698116  -5.1248628   0.4752229
  -2.6115785  -4.1487128   4.6867242  -3.1910141  -5.0885797   3.2708873
   4.7097270  -2.1230593   1.8502713   3.8683531  -3.4108025   1.8591750
   4.3147726  -1.4763184   0.6059915   4.2839609  -1.4361944   3.0621879
   5.7872839  -2.3231987   1.8664360   4.4846674  -4.3169586   1.8811708
   3.1097811  -3.3218621   0.6178520   3.0766763  -3.2835180   3.0754818
   2.3066067  -4.4390168   0.1402191   2.3128133  -4.4111507   3.5928928
   3.3613364  -2.1460998   3.7847760   3.4224283  -2.2144793  -0.1245953
   2.9331232  -1.8589942   4.8892178   3.0341943  -1.9687478  -1.2536616
   2.2738316  -4.4073600  -0.9549382   2.7735655  -5.3729537   0.4736211
   2.2987945  -4.3549476   4.6874651   2.7976384  -5.3403339   3.2719427
   4.1467425   2.9406096   1.8274435   4.8719770   1.5838165   1.8321689
   3.3439056   2.8844915   0.6132129   3.3892819   2.9171965   3.0727101
   4.8404873   3.7889075   1.8038924   5.9627855   1.6888280   1.8086046
   4.3801593   0.9367478   0.6232215   4.4306388   0.9742725   3.0797282
   4.9977936  -0.2845263   0.1230017   5.0090247  -0.2735917   3.5586418
   3.6116542   1.7883832   3.8156311   3.5206232   1.7252810  -0.0955362
   3.2144509   1.5787603   4.9488404   3.0630644   1.4755083  -1.1972748
   4.9598891  -0.2754711  -0.9724059   6.0396600  -0.3163360   0.4620336
   4.9769341  -0.2800230   4.6542571   6.0476720  -0.3332031   3.2135275
   0.0145983  -0.0630565  -0.0707165  -0.9013708  -0.6446731  -0.2316735
   0.8743189  -0.7236095  -0.2372579   0.0478790   0.7373425  -0.8200039
   0.0448335   0.5187684   1.3538053   0.9545738   1.1178295   1.4830317
  -0.0017101  -0.5924721   2.4225144  -0.8071074   1.1964901   1.4883044
  -0.9114047  -1.1915141   2.2928838   0.8501343  -1.2702717   2.2877980
   0.0284301  -0.0116002   3.8474714  -0.8309082   0.6494080   4.0141619
  -0.0057681  -0.8128514   4.5958114   0.9448805   0.5690409   4.0091961
   0.0006010  -0.0001455  -6.0015420  -0.0005356  -0.0015601  -9.0000673
  -0.0001168   2.2000035 -11.2000229
