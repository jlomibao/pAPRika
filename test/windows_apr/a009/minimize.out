
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:05:25

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/a009
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   212386 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  defa
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -8.1356E+02     1.0057E+01     2.4770E+01     N12        57

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -67.7110  EEL     =     1326.2731  EGB        =     -123.8292
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -813.5649
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.2743E+02     4.9770E-01     1.6665E+00     H7        120

 BOND    =        1.2509  ANGLE   =      107.5594  DIHED      =       52.5873
 VDWAALS =      -68.4400  EEL     =     1332.0567  EGB        =     -127.3775
 1-4 VDW =        5.4825  1-4 EEL =    -2130.5545  RESTRAINT  =        0.0065
 EAMBER  =     -827.4354
 NMR restraints: Bond =    0.002   Angle =     0.004   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.2745E+02     4.9334E-01     1.5808E+00     H32       105

 BOND    =        1.2572  ANGLE   =      107.5467  DIHED      =       52.6127
 VDWAALS =      -68.5308  EEL     =     1332.3069  EGB        =     -127.4154
 1-4 VDW =        5.5041  1-4 EEL =    -2130.7471  RESTRAINT  =        0.0111
 EAMBER  =     -827.4656
 NMR restraints: Bond =    0.003   Angle =     0.008   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.2747E+02     4.9677E-01     1.5807E+00     H32       105

 BOND    =        1.2502  ANGLE   =      107.5450  DIHED      =       52.6137
 VDWAALS =      -68.5576  EEL     =     1332.3488  EGB        =     -127.4063
 1-4 VDW =        5.5100  1-4 EEL =    -2130.7888  RESTRAINT  =        0.0137
 EAMBER  =     -827.4850
 NMR restraints: Bond =    0.003   Angle =     0.010   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.2748E+02     4.9085E-01     1.5808E+00     H32       105

 BOND    =        1.2635  ANGLE   =      107.5421  DIHED      =       52.6141
 VDWAALS =      -68.5729  EEL     =     1332.3972  EGB        =     -127.4349
 1-4 VDW =        5.5095  1-4 EEL =    -2130.8167  RESTRAINT  =        0.0155
 EAMBER  =     -827.4982
 NMR restraints: Bond =    0.003   Angle =     0.011   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.2749E+02     4.9031E-01     1.5809E+00     H32       105

 BOND    =        1.2640  ANGLE   =      107.5410  DIHED      =       52.6150
 VDWAALS =      -68.5836  EEL     =     1332.4127  EGB        =     -127.4361
 1-4 VDW =        5.5107  1-4 EEL =    -2130.8309  RESTRAINT  =        0.0170
 EAMBER  =     -827.5072
 NMR restraints: Bond =    0.004   Angle =     0.012   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.2749E+02     5.1456E-01     1.5811E+00     H32       105

 BOND    =        1.2931  ANGLE   =      107.5370  DIHED      =       52.6167
 VDWAALS =      -68.5914  EEL     =     1332.4717  EGB        =     -127.4922
 1-4 VDW =        5.5065  1-4 EEL =    -2130.8534  RESTRAINT  =        0.0182
 EAMBER  =     -827.5119
 NMR restraints: Bond =    0.004   Angle =     0.013   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.2750E+02     4.8971E-01     1.5810E+00     H32       105

 BOND    =        1.2644  ANGLE   =      107.5396  DIHED      =       52.6172
 VDWAALS =      -68.5953  EEL     =     1332.4265  EGB        =     -127.4359
 1-4 VDW =        5.5120  1-4 EEL =    -2130.8448  RESTRAINT  =        0.0190
 EAMBER  =     -827.5163
 NMR restraints: Bond =    0.004   Angle =     0.013   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.2750E+02     4.8971E-01     1.5810E+00     H32       105

 BOND    =        1.2644  ANGLE   =      107.5396  DIHED      =       52.6172
 VDWAALS =      -68.5953  EEL     =     1332.4267  EGB        =     -127.4359
 1-4 VDW =        5.5120  1-4 EEL =    -2130.8451  RESTRAINT  =        0.0190
 EAMBER  =     -827.5164
 NMR restraints: Bond =    0.004   Angle =     0.013   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.2750E+02     4.8971E-01     1.5810E+00     H32       105

 BOND    =        1.2644  ANGLE   =      107.5396  DIHED      =       52.6172
 VDWAALS =      -68.5953  EEL     =     1332.4267  EGB        =     -127.4359
 1-4 VDW =        5.5120  1-4 EEL =    -2130.8450  RESTRAINT  =        0.0190
 EAMBER  =     -827.5164
 NMR restraints: Bond =    0.004   Angle =     0.013   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2750E+02     4.8971E-01     1.5810E+00     H32       105

 BOND    =        1.2644  ANGLE   =      107.5396  DIHED      =       52.6172
 VDWAALS =      -68.5953  EEL     =     1332.4267  EGB        =     -127.4359
 1-4 VDW =        5.5120  1-4 EEL =    -2130.8451  RESTRAINT  =        0.0190
 EAMBER  =     -827.5165
 NMR restraints: Bond =    0.004   Angle =     0.013   Torsion =     0.001
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2750E+02     4.8971E-01     1.5810E+00     H32       105

 BOND    =        1.2644  ANGLE   =      107.5396  DIHED      =       52.6172
 VDWAALS =      -68.5953  EEL     =     1332.4267  EGB        =     -127.4359
 1-4 VDW =        5.5120  1-4 EEL =    -2130.8451  RESTRAINT  =        0.0190
 EAMBER  =     -827.5165
 NMR restraints: Bond =    0.004   Angle =     0.013   Torsion =     0.001
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.304    9.302    0.003    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.268  147.398    0.129    0.001 a
  DUM  DM2    4 --  DUM  DM1    3:   49.490   49.483    0.007    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   61.996   62.003    0.006    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.054 -146.178    0.124    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.723  -72.548    0.175    0.001 t
  DUM  DM1    3 --  C    BUT    2:    5.931    6.000    0.069    0.004 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.355  180.000    0.645    0.002 a
  DUM  DM1    3 --  C3   BUT    2:  178.638  180.000    1.362    0.010 a
                                       Total distance penalty:      0.004
                                       Total angle    penalty:      0.013
                                       Total torsion  penalty:      0.001
|                               RMS deviation from ideal bonds :      0.0096
|                               RMS deviation from ideal angles:      4.465
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.40 (22.60% of Gen B)
|                Calc gb diag               4.18 (39.32% of Gen B)
|                Calc gb off-diag           4.05 (38.06% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time             10.64 (100.0% of Nonbo)
|          Nonbond force             10.64 (90.63% of Force)
|          Bond/Angle/Dihedral        1.06 ( 9.06% of Force)
|          Other                      0.04 ( 0.30% of Force)
|       Force time                11.74 (100.0% of Runmd)
|    Runmd Time                11.74 (99.76% of Total)
|    Other                      0.03 ( 0.24% of Total)
| Total time                11.77 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:05:25.209  on 04/10/2019
|           Setup done at 18:05:25.213  on 04/10/2019
|           Run   done at 18:05:36.978  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
